lib/pages/my_reports_pages/components/reports_list_widget.dart

```diff
@@ -57,22 +58,19 @@ class ReportsList extends StatelessWidget {
                             crossAxisAlignment: CrossAxisAlignment.start,
                             children: <Widget>[
                               Style.titleMedium(
-                                  MyLocalizations.of(
-                                          context, 'report_of_the_day_txt')! +
-                                      DateFormat('dd-MM-yyyy')
-                                          .format(DateTime.parse(
-                                              reports[index].creation_time!))
-                                          .toString(),
+                                  Utils.getTranslatedReportType(context, reports[index].type),
                                   fontSize: 14),
                               Style.body(
                                   '${MyLocalizations.of(context, "location_txt")} ${reports[index].displayCity ?? ''}'),
```

Comment:
Add `:` after the location_txt translated string

Reply:
should I remove the `:` from the strings where it's included, like ca, es, eu, gl, pt? I am afraid there would be a double `:` otherwise

Reply:
I've already removed them from the localise.biz platform. So in the next export, it will be already removed :)